containers/src/Data/Sequence/Internal.hs

```diff
@@ -395,33 +394,45 @@ fmapSeq f (Seq xs) = Seq (fmap (fmap f) xs)
  #-}
 #endif
 
-getSeq :: Seq a -> FingerTree (Elem a)
-getSeq (Seq xs) = xs
-
 instance Foldable Seq where
-    foldMap f = foldMap (f .# getElem) .# getSeq
-    foldr f z = foldr (f .# getElem) z .# getSeq
-    foldl f z = foldl (f .^# getElem) z .# getSeq
+#ifdef __GLASGOW_HASKELL__
```

Comment:
We need `foldMap` for other implementations too.

Reply:
What do you mean? We have `foldMap` implementations.

Reply:
Am I missing something? It looks like you just hid it behind an `ifdef`.

Reply:
Oh, you mean _Haskell_ implementations, I misunderstood. Those definitions are in the `#else` part of this `#ifdef`.

Reply:
Does it seem fine, @treeowl?